docs/releases/5.0.txt

```diff
@@ -45,6 +45,22 @@ toggled on via the UI. This behavior can be changed via the new
 :attr:`.ModelAdmin.show_facets` attribute. For more information see
 :ref:`facet-filters`.
 
+Database defaults
+-----------------
```

Comment:
The phrase "database default" might sound like a "default database", rather than a field value..

```suggestion
Database-level default values
-----------------------------
```